include/solution_transfer/meshless_interpolation_function.h

```diff
@@ -29,126 +27,55 @@
 #include <cstddef>
 #include <memory>
 
-namespace libMesh
-{
-
-
+namespace libMesh {
 
 // Forward Declarations
-template <typename T>
-class DenseVector;
-
+template <typename T> class DenseVector;
 
 // ------------------------------------------------------------
 // MeshlessInterpolationFunction class definition
-class MeshlessInterpolationFunction : public FunctionBase<Number>
-{
+class MeshlessInterpolationFunction : public FunctionBase<Number> {
 private:
-  const MeshfreeInterpolation & _mfi;
+  const MeshfreeInterpolation &_mfi;
```

Comment:
Don't delete this space.

Reply:
at some point i deactivated the clang formatting that edits the code when i save. But maybe some edits got through before that. Is there something equivalent for the libmesh world?

Reply:
I think @lindsayad was going to throw a .clang-format file into the libmesh repo at some point, but it doesn't look like he did so yet. Other than that, we just have some [formatting conventions](https://github.com/libMesh/libmesh/wiki/Source-code-formatting-conventions) that we use, but aren't rigidly enforced by anything.

Reply:
Yea I haven't thrown anything in yet, to my chagrin

Reply:
I tried to throw in one once, but it kind of backfired on me as soon as I started saving files with my vim "auto-apply .clang-format" settings.  clang-format doesn't have (and IIRC has a FAQ about not having) an option to tell it to enforce rules A1, B3, and C2 while simply preserving the existing formatting for rules D through Z; it's an all-or-nothing reformatter.

If we want to decide on formatting for every option they have, I guess we could do all the changes in one giant auto-reformat-everything commit and then proceed from there, but that's the only way to avoid editors turning every `:wq` into a giant mess of whitespace changes.

Reply:
I tell emacs to exclude certain directories like emacs for my clang formatting hook. I would assume you could tell vim the same thing, but if not, then I guess emacs must be better 😺

Reply:
:fishing_pole_and_fish:

I can tell vim to ignore libMesh directories, but then what's the point of having a .clang-format file there?  Just to look pretty?

Reply:
What I do when developing new code in libMesh is to write it, then select the new code region, and then `M-x clang-format`